### ISSUE TITLE
Digital amputation

### ISSUE THREAD

**Issue by carolina-f** [obophenotype/human-phenotype-ontology#10961]
**Preferred term label:**
See issue #10960

**Synonyms**
toe amputation
finger amputation

**Definition (free text, please give PubMed ID)**
PMID: 31424867 Constriction rings and **limb or digital amputation** are the most common findings, present in at least 80 percent of cases. The amniotic bands may be present, confined to the skin or soft tissue, and extend deep into the tissue. If constriction results in **amputation in utero**, then the amputated part is usually resorbed and not visible following birth.

PMID: 31407870 The phenotypic presentation in the extremities has considerable variation which can range from simple constriction rings to more complex scenarios. This includes circumferential constriction rings, which can lead to distal ischemia resulting in significant edema and/or limb and digital amputations.

PMID: 31787161 A constellation of multiple fetal abnormalities, particularly asymmetric limb and digital amputations and constrictions, should make one suspect amniotic band sequence.

**Parent term (use hpo.jax.org/app)**
Congenital amputation issue #10960

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
OMIM 217100 CONSTRICTING BANDS, CONGENITAL
ORPHA:295000 Amniotic band syndrome

**Your nano-attribution (ORCID)**
0009-0005-6714-5727

**Comment by pnrobinson** [comment]
@carolina-f I think this is : [Autoamputation of digits HP:0007460](https://hpo.jax.org/browse/term/HP:0007460). 
According to my Sprachgefühl, Amputation is by an external process, and Autoamputation refers to the process described above. Thoughts?

**Comment by pnrobinson** [comment]
@carolina-f - so is Autoamputation of digit correct for this annotation?

**Comment by carolina-f** [comment]
@pnrobinson According to the definition of autoamputation in HPO: Auto-amputation is the spontaneous detachment of an appendage from the body due to long standing pathology.
In the case of intrauterine amputations, there are three main theories of the etiology of an amniotic constriction band, including the intrinsic theory, extrinsic theory, and vascular theory. (PMID: 32954908)
The intrinsic theory was presented by Streeter in 1930, who considered that both anomalies and fibrous bands originated
from perturbation of a developing germinal disc in the early embryo. This theory is usually used to account for major craniofacial
abnormalities, body wall defects, and internal abnormalities. The extrinsic theory was proposed by Torpin in 1965, who suggested that rupture of the amniotic sac is responsible for the formation of amniochorionic mesodermal bands leading to development of an amniotic constriction band. The vascular theory was described by Van Allen in 1981, who claimed that internal and external defects can result from vascular disruption.
I am not sure that any of these 3 theories correspond to a long-term pathology.
Although autoamputation is often regarded as an acquired ailment, it can also be congenital. Therefore, the distinction between **congenital** (or **intrauterine) autoamputation** and acquired autoamputation could be more accurate.

**Comment by pnrobinson** [comment]
@carolina-f thanks!
Adding new term: 
- ID: HP:6001390
- label: Intrauterine digital amputation
- parents: [Abnormal fetal skeletal morphology (HP:0025662)](https://hpo.jax.org/app/browse/term/HP:0025662)
- pmid: 
- def: A developmental anomaly characterized by partial or complete absence of a finger or toe at birth, resulting from interruption of normal development in utero.
- comment: 
- orcid: orcid:0009-0005-6714-5727
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10961
- synonyms:
